charts/connectors/values.go

```diff
@@ -0,0 +1,242 @@
+	Deployment     Deployment     `json:"deployment"`
+	Storage        Storage        `json:"storage"`
+	ServiceAccount ServiceAccount `json:"serviceAccount"`
+	Service        Service        `json:"service"`
+}
+
+type Image struct {
+	Repository string            `json:"repository"`
+	PullPolicy corev1.PullPolicy `json:"pullPolicy"`
+	Tag        string            `json:"tag"`
+}
+
+type Connectors struct {
+	RestPort                int32             `json:"restPort"`
+	BootstrapServers        string            `json:"bootstrapServers"`
+	SchemaRegistryURL       string            `json:"schemaRegistryURL"`
+	AdditionalConfiguration string            `json:"additionalConfiguration"`
+	SecretManager           SecretManager     `json:"secretManager"`
+	ProducerBatchSize       int32             `json:"producerBatchSize"`
+	ProducerLingerMS        int32             `json:"producerLingerMS"`
+	Storage                 ConnectorsStorage `json:"storage"`
+	GroupID                 string            `json:"groupID"`
+	BrokerTLS               TLS               `json:"brokerTLS"`
+}
+
+type SecretManager struct {
+	Enabled          bool   `json:"enabled"`
+	Region           string `json:"region"`
+	ConsolePrefix    string `json:"consolePrefix"`
+	ConnectorsPrefix string `json:"connectorsPrefix"`
+}
+
+type ConnectorsStorage struct {
+	ReplicationFactor struct {
+		Offset int32 `json:"offset"`
+		Config int32 `json:"config"`
+		Status int32 `json:"status"`
+	} `json:"replicationFactor"`
+	Remote struct {
+		Read struct {
+			Offset bool `json:"offset"`
+			Config bool `json:"config"`
+			Status bool `json:"status"`
+		} `json:"read"`
+		Write struct {
+			Offset bool `json:"offset"`
+			Config bool `json:"config"`
+			Status bool `json:"status"`
+		} `json:"write"`
+	} `json:"remote"`
+	Topic struct {
+		Offset string `json:"offset"`
+		Config string `json:"config"`
+		Status string `json:"status"`
+	} `json:"topic"`
+}
+
+type TLS struct {
+	Enabled bool `json:"enabled"`
+	CA      struct {
+		SecretRef           string `json:"secretRef"`
+		SecretNameOverwrite string `json:"secretNameOverwrite"`
+	} `json:"ca"`
+	Cert struct {
+		SecretRef           string `json:"secretRef"`
+		SecretNameOverwrite string `json:"secretNameOverwrite"`
+	} `json:"cert"`
+	Key struct {
+		SecretRef           string `json:"secretRef"`
+		SecretNameOverwrite string `json:"secretNameOverwrite"`
+	} `json:"key"`
+}
+
+type Auth struct {
+	SASL struct {
+		Enabled   bool   `json:"enabled"`
+		Mechanism string `json:"mechanism"`
+		SecretRef string `json:"secretRef"`
+		UserName  string `json:"userName"`
+	} `json:"sasl"`
+}
+
+type Logging struct {
+	Level string `json:"level"`
+}
+
+type Monitoring struct {
+	Enabled           bool              `json:"enabled"`
+	ScrapeInterval    Duration          `json:"scrapeInterval"`
+	Labels            map[string]string `json:"labels"`
+	Annotations       map[string]string `json:"annotations"`
+	NamespaceSelector struct {
```

Comment:
nit: there's a type for this in the `monitoringv1` package that can be found in the imports of redpanda's values.go. The struct is otherwise correct though!